content/docs/install/completion.md

```diff
@@ -27,12 +27,13 @@ run     -- Generate a stage file from a command and execute the command.
 
 Depending on what you typed on the command line so far, it completes:
 
-- Available DVC commands.
+- Available DVC commands and subcommands. For example, `dvc plots` completes
+  with `diff`, `modify`, and `show`.
 - Options (flags) that are available for a particular command.
 - File names that make sense in a given context, such as using them as a target
   for some commands.
 - Values for certain command arguments. For example, `dvc repro` completes with
-  stage files to reproduce.
+  stages or `.dvc` files to reproduce.
```

Comment:
I highly doubt the autocompletion tool reads stages from dvc.yaml files. It probably only works with .dvc files. In fact it probably completes any file in the cwd. Please try it. Editing most of these docs usually involves actually trying the tool in question. Thanks

Reply:
> OK. I see you mentioned trying the tool here. Yeah it's not that smart probably. You can just say something like "existing files" here.

Yes, this was the motivation for replacing the `dvc repro` example in the first place. It didn't complete with stages from dvc.yaml. It does complete with .dvc files so "existing files" makes sense.